.github/workflows/windows/build/action.yml

```diff
@@ -0,0 +1,30 @@
+name: "Build DLL files for Windows"
+description: "Prepares the PHP build environment for the MongoDB driver"
+inputs:
+  version:
+    description: "PHP version to build for"
+    required: true
+  arch:
+    description: "The architecture to build for (x64 or x86)"
+    required: true
+  ts:
```

Comment:
Do you want to consider using choice and boolean fields for `arch` and `ts`, respectively? I noted that php/setup-php-sdk expects strings for both, so perhaps the choice field would make to most sense (since boolean would require some logic later to derive the correct strings).

See: https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
The reference you linked refers to inputs that are used when triggering a workflow run (as is done for release.yml). This file contains metadata for an action, where inputs are always of type string: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs